src/App.jsx

```diff
@@ -45,17 +55,17 @@ export function App() {
 			/** Finally, we update our React state. */
 			setData(nextData);
 		});
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [listToken]);
 
 	return (
-		<Router>
-			<Routes>
-				<Route path="/" element={<Layout />}>
-					<Route index element={<Home />} />
-					<Route path="/list" element={<List data={data} />} />
-					<Route path="/add-item" element={<AddItem />} />
-				</Route>
-			</Routes>
-		</Router>
+		<Routes>
```

Comment:
I like this refactor and cleaning up all these nested routes a little bit and moving the Router to index.js ✨ Great focus on proactively making the code more readable and the codebase grows!